Formula/eris.rb

```diff
@@ -0,0 +1,24 @@
+class Eris < Formula
+  desc "Eris blockchain application platform CLI"
+  homepage "https://erisindustries.com"
+  url "https://github.com/eris-ltd/eris-cli/archive/v0.11.4.tar.gz"
+  sha256 "e2eb02d01b76e8be9f28aac31b2e56ebadc4d0decb21fcfefb2f219cb03d1238"
+
+  depends_on "go" => :build
+  depends_on "docker"
+  depends_on "docker-machine"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = bin
+
+    (buildpath/"src/github.com/eris-ltd").mkpath
+    ln_sf buildpath, buildpath/"src/github.com/eris-ltd/eris-cli"
+
+    system "go", "build", "-o", "#{bin}/eris", "github.com/eris-ltd/eris-cli/cmd/eris"
+  end
+
+  test do
+    system "#{bin}/eris", "version"
```

Comment:
This is failing: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/1894/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_eris/